src/components/staking/PageValidator.vue

```diff
@@ -295,32 +271,18 @@ export default {
     },
     status() {
       if (this.validator.jailed) return `Jailed`
-
-      if (parseFloat(this.validator.status) === 0) return `Inactive`
-
+      else if (this.validator.status === 0) return `Inactive`
       return `Active`
     },
-    statusColor() {
-      if (this.validator.jailed) return `red`
-
-      if (parseFloat(this.validator.status) === 0) return `orange`
-
-      return `green`
-    },
-    // empty descriptions have a strange '[do-not-modify]' value which we don't want to show
     website() {
```

Comment:
We don't want to show `[do-not-modify]` which is/was a default value and shows in some cases

Reply:
I caught this with the noBlanks filter (not the best name)

but… I don't think it works well for website. I'll fix this.